proto/viam/app/v1/app.proto

```diff
@@ -269,11 +275,13 @@ message LogEntry {
 
 message GetRobotPartLogsResponse {
   repeated LogEntry logs = 1;
+  optional pageToken next_page_token = 5;
 }
 
 message TailRobotPartLogsRequest {
   string id = 1;
   bool errors_only = 2;
+  optional string filter = 3;
```

Comment:
not sure if we actually want this -- should we make it s.t. this endpoint only ever returns live functions without a filter? Thinking about it more that makes more sense to me

Reply:
I think it makes sense to have it. We will want a filter with tail eventually.